internal/compiler/passes/binding_analysis.rs

```diff
@@ -161,6 +167,17 @@ fn analyze_element(
     for (_, nr) in &elem.borrow().accessibility_props.0 {
         process_property(&PropertyPath::from(nr.clone()), context, reverse_aliases, diag);
     }
+    if elem.borrow().repeated.is_some() {
+        if let ElementType::Component(base) = &elem.borrow().base_type {
+            if base.parent_element.upgrade().is_some() {
```

Comment:
why this test?

Reply:
That seems indeed unnecessary, here and in a few other places. I'll add a patch to remove that.

Reply:
Looks like it's not that easy - in `recurse_elem_including_sub_components` that check seems needed. I'll try to find out why.